rewards/df-challengedf.md

```diff
@@ -0,0 +1,47 @@
+---
+description: >-
+  Weekly prediction challenges for data scientists to compete, launch algorithms, and start earning. 🏆
+---
+
+# Win Competitions in Challenge DF
+
+<figure><img src="../.gitbook/assets/rewards/gif/eat-data.gif" alt=""><figcaption><p>Let's consume some data.</p></figcaption></figure>
+
+## Publish prediction models and earn rewards
+
+If you're eager to consume data and create valuable assets for publishing but unsure of where to begin, fear not! We've got you covered.
+
+Data Farming offers rewards to data scientists who participate in weekly challenges. We provide the rules and rewards, allowing you to focus on delivering high-quality results.  
+
+**Prizes:**
+1st Place - 2500 OCEAN
+2nd Place - 1500 OCEAN
+3rd Place - 1000 OCEAN
+_Total - 5000 OCEAN_
+
+### What are the base rules?
+
+- A fresh challenge begins and concludes every week.
+- At the beginning of each week, a predetermined amount of OCEAN tokens is allocated as rewards.
+- The rewards are divided into specific amounts for the top three positions.
+- The individuals who submit the top three entries will receive rewards, and their names will be showcased within the Data Farming dapp.
+
+### What do I gain from this?
+
+- Earn OCEAN rewards, [claimable inside our app](https://df.oceandao.org/rewards)
+- Gain in-depth knowledge of the Ocean Protocol stack.
+- Increase your visibility by achieving a place on the podium.
+- Create valuable assets that can be monetized on the Ocean Marketplace.
+- Enhance your skills and engage in friendly competition with other participants.
+
+### Where to start?
+
+Head over to the [Challenges Page](https://df.oceandao.org/challenges) to learn more about Challenge DF, how to submit, and to see historical results.
+
+If you're looking for a step-by-step guide on everything from understanding what a challenge entails to claiming your well-deserved rewards, be sure to check out [how to farm Challenge DF rewards](../user-guides/data-farming/how-to-challengedf.md). It will provide you with all the necessary information you need to make the most out of the challenges.
+
+For a direct link to the Challenge DF README, [click here](https://github.com/oceanprotocol/predict-eth/blob/main/challenges/challenge-df.md).
+
+### Good lock and happy coding!
+
+<figure><img src="../.gitbook/assets/rewards/gif/winner.gif" alt=""><figcaption></figcaption></figure>
```

Comment:
The image is not loaded....I'll investigate this as well

Reply:
Please see comments below & use the 2nd gitbook link.
I have reviewed and it seems to be working, resolving this convo.